test/com/nikhaldimann/inieditor/IniEditorTest.java

```
@@ -306,17 +327,17 @@ public void testSave() throws IOException {
      */
     public void testSaveLoadCharset() throws IOException {
        	String[] expected = new String[] {
-       	    "[c�mm�n]", "[t�st]", "", "h�llo = vel�"
+       	    "[c�mm�n]", "[t�st]", "", "h�llo = vel�"
```

This was unintentional. Intellij warned that the file was not encoded in UTF-8 and I ignored the error but it must have still changed the encoding.